tests/utils/pvc.go

```diff
@@ -213,7 +213,7 @@ func NewBlockPVCDefinition(pvcName string, size string, annotations, labels map[
 func WaitForPersistentVolumeClaimPhase(clientSet *kubernetes.Clientset, namespace string, phase k8sv1.PersistentVolumeClaimPhase, pvcName string) error {
 	err := wait.PollImmediate(pvcPollInterval, pvcPhaseTime, func() (bool, error) {
 		pvc, err := clientSet.CoreV1().PersistentVolumeClaims(namespace).Get(context.TODO(), pvcName, metav1.GetOptions{})
-		fmt.Fprintf(ginkgo.GinkgoWriter, "INFO: Checking PVC phase: %s\n", string(pvc.Status.Phase))
+		fmt.Fprintf(ginkgo.GinkgoWriter, "INFO: Checking PVC phase: %s, resource version %s\n", string(pvc.Status.Phase), pvc.ResourceVersion)
```

Comment:
Should we add a `Consistently()` block to the failing test, that ensures the PVC doesn't get updated from a certain point onwards?

Reply:
Can you elaborate, I don't quite understand what you mean

Reply:
I mean we are missing a functional test that ensures we are not hotlooping PVC updates

Reply:
Yeah, we are, just need to identify how to reliably test that. As we saw the revisionVersion goes up relatively fast even if we are not hot looping